dwave/cloud/client.py

```diff
@@ -1600,29 +1637,75 @@ def upload_problem_encoded(self, problem, problem_id=None):
             self._upload_problem_worker, problem=problem, problem_id=problem_id)
 
     @staticmethod
-    def _parse_multipart_upload_response(response):
-        """Parse the JSON response body, raising appropriate exceptions."""
+    def _sapi_request(meth, *args, **kwargs):
+        """Execute an HTTP request defined with the ``meth`` callable and
+        parse the response and interpret errors in compliance with SAPI REST
+        interface.
+
+        Note:
+            For internal use only.
+
+        Args:
+            meth (callable):
+                Callable object to be called with args and kwargs supplied, with
```

Comment:
This seems to state the code. Maybe something like, "The HTTP request to execute. Behavior must be consistent ..."